client/homescreen/layout.js

```diff
@@ -83,6 +83,7 @@ export const Layout = ( {
 	}, [ maybeToggleColumns ] );
 
 	const shouldStickColumns = isWideViewport.current && twoColumns;
+	const shouldShowStoreLinks = taskListComplete || ! isTaskListEnabled;
```

Comment:
Would it still be worth to keep the `isTaskListHidden` and add it onto this `||` case?
That way if the user hides the task list without completing it, it will still show the store management links.

(side note, kinda unrelated) `isTaskListEnabled` kinda confused me, I guess just means if both task lists are hidden (but that might never happen if the user doesn't have a plugin with custom tasks).

Reply:
@louwie17 yeah in testing isTaskListEnabled seemed fine without custom tasks, but I agree it might make more sense to check `isTaskListHidden` instead. I'll investigate this.

Reply:
I have resolved this by just using `isTaskListHidden` here: https://github.com/woocommerce/woocommerce-admin/pull/6657/commits/a568e6614fa81cb4abdceaea46f51b6ad4e97a8d tests well for me. I don't think we need both since `isTaskListEnabled` is determined by `isTaskListHidden` and another specific option. 

Thanks for suggesting this, I think its a lot better.